frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -235,6 +254,7 @@ export class Dashboard extends Component {
       this.setState({
         lat: Number(value.latitude),
         long: Number(value.longitude),
+        zoom: 14,
```

Comment:
Can you assign these 14s and 6s to a `const` at the top of the file so that they're not just magic numbers

Reply:
Done